crates/libtiny_client/src/stream.rs

```diff
@@ -14,22 +14,53 @@ use tokio_native_tls::TlsStream;
 #[cfg(feature = "tls-rustls")]
 use tokio_rustls::client::TlsStream;
 
+use crate::SASLExternal;
+
 #[cfg(feature = "tls-native")]
 lazy_static! {
-    static ref TLS_CONNECTOR: tokio_native_tls::TlsConnector =
-        tokio_native_tls::TlsConnector::from(native_tls::TlsConnector::builder().build().unwrap());
+    static ref TLS_CONNECTOR: tokio_native_tls::TlsConnector = tls_connector(None);
+}
+
+#[cfg(feature = "tls-native")]
+fn tls_connector(sasl: Option<&SASLExternal>) -> tokio_native_tls::TlsConnector {
+    let mut builder = native_tls::TlsConnector::builder();
+    if let Some(SASLExternal { cert, key }) = sasl {
+        todo!("Waiting for https://github.com/sfackler/rust-native-tls/pull/209")
```

Comment:
Hm, I missed this part. We should wait for this PR, right? Would be good to mention this kind of thing as the first thing in the PR description as I often don't review line by line and miss things.

Reply:
Yes, sorry about that. Let hold until it's resolved. I suppose we _could_ merge (without a release) and then I will address the `todo!()` in another PR once the issue is resolved, but it's fine to just wait.

Reply:
This is completed now